specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2022-10-01-preview/ThreatIntelligence.json

```diff
@@ -464,6 +464,96 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.OperationalInsights/workspaces/{workspaceName}/providers/Microsoft.SecurityInsights/threatIntelligenceIndicators/query": {
```

Comment:
>threatIntelligenceIndicators/query [](http://example.com/codeflow?start=176&length=34)

POST action is not allowed on a resource type. It needs to be on individual resource or at tenant level, e.g.

/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.OperationalInsights/wokspaces/{workspaceName}/providers/Microsoft.SecurityInsights/listThreatIntelligenceIndicators

Reply:
Done thanks.